<!DOCTYPE html>
<html>
    <head>
        <title>
            Team:Stony_Brook/Design - 2020.igem.org
        </title>
        <link rel = "stylesheet" href="assets/css/implementation.css">
        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
         <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
         <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
          <!--   <link rel = "stylesheet" href = "style.scss"> -->
         <!-- If you want special fotns, use link href
         = "url of font" and then insert the font family from online-->
    </head>
    <body>
        <nav class="navbar navbar-purple navbar-expand-md py-4" style="margin-bottom: 0px; border-radius: 0px;">

            <a class="navbar-brand">Light Switch</a>

            <ul class="nav navbar-nav navbar-right" style="padding-right: 15px; font-size: large;">
                <li><a href="index.html">Home</a></li>
                <li class="nav-item dropdown">
                    <a href="#" role="button" aria-haspopup="true">
                        Team
                        <span class="caret"></span>
                    </a>
                    <ul class="dropdown-menu">
                        <li><a href="Team Members.html">Team Members</a></li>
                        <li><a href="Collaborations.html">Collaborations</a></li>
                        <li><a href="Attributions.html">Attributions</a></li>
                    </ul>
                </li>

                <li class="nav-item dropdown">
                    <a href="#" role="button" aria-haspopup="true">
                        Project
                        <span class="caret"></span>
                    </a>
                    <ul class="dropdown-menu">
                        <li><a href="Description.html">Description</a></li>
                        <li><a href="Design.html"> Design</a></li>
                        <li><a href="Experiments.html"> Experiments</a></li>
                        <li><a href="Notebook.html"> Notebook</a></li>
                        <li><a href="Modeling.html"> Modeling</a></li>
                        <li><a href="Engineering.html">Engineering</a></li>
                        <li><a href="Implementation.html">Implementation</a></li>
                        <li><a href="Contributions.html">Contributions</a></li>
                    </ul>
                </li>
                <li><a href="Safety.html">Safety</a></li>

                <li class="nav-item dropdown">
                    <a href="Human Practices.html" role="button" aria-haspopup="true">
                        Human Practices
                        <span class="caret"></span>
                    </a>
                    <ul class="dropdown-menu">
                        <li><a href="Education and Engagement.html">Education and Engagement</a></li>
                        <li><a href="Integrated.html">Integrated Human Practices</a></li>
                    </ul>
                </li>

                <li class="nav-item dropdown">
                    <a href="Awards.html" role="button" aria-haspopup="true">
                        Awards
                        <span class="caret"></span>
                    </a>
                    <ul class="dropdown-menu">
                        <li><a href="Plant.html">Plant</a></li>
                        <li><a href="Medals.html">Medals</a></li>
<li><a href="Excellence.html">Excellence</a></li>
                    </ul>
                </li>
            </ul>
        </nav>

        <header class="header-container">
            <img src="https://2020.igem.org/wiki/images/7/74/T--Stony_Brook--design.jpg" alt="" style="width:100%;height:auto;opacity:0.5;">
            <!--CSS for the loading page -->
            <div class="centered"> Design </div>
        </header>
              <br>

              <div class="text-container" style="display: inline-block;">
                  <p style="font-size: 35px;"> <b>Overview</b> </p>
                  <hr class="hr">
                   <p style="font-size: 18px;">
                        For many decades, the agricultural industry has sought to improve crop yields and mitigate crop losses due to pests and pathogens by utilizing GM crops. However, an increased use of GM crops can negatively impact biodiversity through gene flow, potentially allowing GM crops to out compete their wild-type counterparts (Carpenter).To address this concern, we have created an optogenetic kill-switch capable of initiating programmed cell death (PCD) if a plant were to escape from its controlled indoor setting to the outdoor environment. A light-mediated solution offers an unprecedented way to control cellular behavior with precise spatial and temporal resolution; this is difficult with alternative methods, which often utilize toxic chemical inducers. Our proposed system would allow farmers to reap the benefits of GM crops without worrying about their harmful ecological implications.
                   </p>
                   <p style = "font-size: 18px;">
                    Currently, indoor farming works to mitigate the risk of gene flow by providing some degree of control over the plants being grown. However, it has no inherent mechanism preventing the escape of transgenes. In order to issue direct control of plant growth, manipulation of the shoot apical meristem (SAM) may be desired. SAMs are the source of above-ground organs and can be classified into different zones based on cytology (Somssich et al. 2016). The central zone (CZ) contains a pool of pluripotent stem cells which divide slowly and replace the daughter cells in the peripheral zone (PZ) (Figure 1). These daughter cells, which have a higher rate of cell division, form the organ primordia on the flanks of the SAM. A small group of cells underneath the CZ, the organizing center (OC), expresses the transcription factor WUSCHEL (WUS).
                </p>
                <br>
                <div align="center">
                    <figure>
                        <img src="https://2020.igem.org/wiki/images/5/5e/T--Stony_Brook--design_figure_1.jpg" style="width:50%; padding-right:10px; padding-bottom:10px;">
                        <figcaption>Figure 1: Structure and composition of the shoot apical meristem (SAM).
                        </figcaption>
                    </figure>
                </div>
                <br>

              </div>
              <div class = "text-container" style = "display: inline-block;">
                   <p style="font-size: 35px;">
                     <b> CLAVATA-WUSCHEL Pathway and syn-tasiRNA</b>
                   </p>
                   <hr class = "hr">
                   <p style="font-size: 18px;">
                    The CLAVATA-WUSCHEL signaling pathway is responsible for maintaining the meristematic stem cell population in the SAM. In Arabidopsis thaliana, the coordination of cell proliferation and differentiation is achieved through an autoregulatory negative feedback loop composed of the genes WUSCHEL (WUS) and CLAVATA3 (CLV3) (Figure 2). CLV3 encodes a signaling peptide that interacts with plasma-membrane localized receptor-like kinases (RLKs) such as CLV1 and CLV2. This triggers a signalling cascade that ultimately downregulates WUS transcription (Adibi et al. 2016). Further research is necessary to accurately characterize the CLAVATA-WUSCHEL pathway in N. benthamiana, but the conservation of the negative feedback loop in plant species such as Arabidopsis, Solanum lycopersicum (tomato), Oryza sativa (rice), and Zea mays (maize) suggest that it is relatively conserved in N. benthamiana (Fletcher 2018).
                    </p>
                <div align = "center">
                    <br>
                    <figure>
                        <img src="https://2020.igem.org/wiki/images/f/fc/T--Stony_Brook--design_figure_2.jpg" style="width:50%; padding-right:10px; padding-bottom:10px;">
                        <figcaption>Figure 2: Representation of the CLV-WUS pathway.
                        </figcaption>
                    </figure>
                    <br>
                </div>
                    <p style = "font-size: 18px;">
                        When WUS expression is reduced, stem cell differentiation is promoted and the SAM stem cell population is depleted. This allows for stem cells to differentiate but prevents them from being replaced, preventing growth of the whole plant. Knockdown of the WUS gene is accomplished through the production and proliferation of synthetic trans-acting small interfering RNAs (syn-tasiRNAs). Syn-tasiRNA biogenesis begins with the transcription of a syn-tasiRNA precursor with RNA Polymerase II. This single-stranded precursor undergoes Argonaute 1 (AGO1)-mediated cleavage guided by a co-expressed miRNA, miR173. The cleaved syntasi-RNA is then acted upon by RNA-dependent RNA polymerase 6 (RDR6), forming a double-stranded RNA which is cleaved by Dicer-like 4 (DCL4). This results in the formation of a mature, double stranded, 21-nt long syntasi-RNA (Allen et al. 2010). This mature syntasi-RNA is then loaded into Argonaute 2 (AGO2), which cleaves the syntasi-RNA passenger strand. The complex of the guide strand and AGO2 then forms the RNA-Induced Silencing Complex (RISC) with the guide strand and mRNA. Within the RISC, the guide strand is used for Watson-Crick base pairing to target mRNA transcript and AGO2 functions as a riboendonuclease that cleaves target mRNA (Carthew et al. 2009). Cleavage of the mRNA transcript obstructs translation, “silencing” the gene. This silencing is not isolated to the cells which produce interfering RNAs. Ta-siRNAs, si-RNAs, and mi-RNAs are highly mobile and can affect gene silencing in distal plant tissues through movement via the phloem.
                    </p>

                    <p style = "font-size: 18px;">
                        The transcription of the WUS syn-tasiRNA is controlled by a light-inducible promoter activated by a UVR8-COP1 optogenetic pair. UVR8 is a plant photoreceptor responsible for regulating UV-B-triggered signalling pathways (Yang et al, 2015). Its binding partner, COP1, is a key regulator of photomorphogenesis. UVR8 perceives light in the UV-B region using tryptophan residues (Trp 233/285) as chromophores. Upon UV-B irradiation UVR8 undergoes a conformational change and completely dissociates, exposing a 27 residue C-terminal extension (C27), facilitating UVR8-COP1 interaction (Figure 3).
                    </p>

                    <p style = "font-size: 18px;">
                        pCOP1-UVR8 and pAtTASI were produced from pFGL815, an empty backbone designed for Agrobacterium-mediated transformation (Yang et al. 2014). pCOP1-UVR8, and pAtTASI inserts encode the optogenetic transcription control system and a syn-tasiRNA directed against WUS mRNA, respectively. N. benthamiana is then transformed through Agrobacterium-mediated infiltration. Upon controlled UV-B exposure, the UVR8-COP1 pair associates, activating expression of syn-tasiRNA.
                    </p>

                    <div align = "center">
                    <br>
                    <figure>
                        <img src="https://2020.igem.org/wiki/images/0/07/T--Stony_Brook--design_figure_3.jpg" style="width:50%; padding-right:10px; padding-bottom:10px;">
                        <figcaption>Figure 3: UV-B light inducible gene expression system. UV-B light-induced monomerization of UVR8 recruits COP1-VP16 to the CaMV core promoter, thus activating the expression of the tasi-RNA precursor.</figcaption>
                    </figure>
                   <br>

                    </div>
                 </div>

                 <div class="text-container">
                    <p style="font-size: 35px;">
                        <b>Constructs</b>
                    </p>
                    <hr class = "hr">
                    <p style="font-size: 18px;">
                      In our design, UVR8 is fused to TetR, otherwise known as the tetracycline repressor domain, which remains bound to the tetO consensus sequence. Though UVR8-TetR is constitutively bound to the tetO site, no transcription occurs while the UVR8-COP1 pair is unbound. The VP16 activation domain, which functions as an activating domain in our design, is fused to COP1. Dissociation in response to UV irradiation exposes the C-terminal 27 (C27) residues. The exposed UVR8 homodimer interface, and more specifically C27, then acts as an available site for the WD40 domain of COP1 to bind to. This results in the recruitment of COP1-VP16 to the nucleus, where it binds to the CaMV core promoter, and induces transcription of the ta-siRNA precursor by RNA polymerase II. The WD40 domain is also capable of binding to the full-length of UVR8, but only in its monomeric form.
                    </p>
                    <br>
                    <div align="center">
                    <img src="https://2020.igem.org/wiki/images/d/d0/T--Stony_Brook--circuit.png" style="width:50%; padding-right:10px; padding-bottom:10px;">
                    </div>
                    <br>
                    <p style="font-size: 18px;">
                      For this design we designed two constructs, both of which used the pFG815 backbone. This backbone was chosen because it was built for agrobacterium-mediated transformation (Yang et al., 2014). It contains 25 bp LB and RB T-DNA repeats which could be recognized by the <i>vir</i> gene on the helper plasmid in <i>Agrobacterium</i> to help facilitate transfer of our construct into the plant’s cells. There is a 69-bp MCS which contains 12 unique restriction enzyme sites to insert our construct into. There is also an origin of replication for both E. coli and Agrobacterium, allowing the plasmid to stably exist in both organisms. The KanR gene confers resistance to kanamycin, which was used to select for transformed XL1 blue competent E. coli and Agrobacterium strain EHA105. We linearized this vector with XmnI and HindIII-HF. Cutting with two restriction enzymes minimizes the chance of spontaneous re-ligation and removes the CaMV poly(A) signal from the original backbone.
                    </p>
                    <p style="font-size: 18px;">
                      Since our vector did not contain a promoter or terminator, we put the genes for the COP1-VP16 and mphR(A)-UVR8 fusion proteins under the control of a 35S CaMV promoter and 35S CaMV terminator in our first construct, pAtTASI-HiFi. These regulatory elements are commonly used in plant synthetic biology and would allow for strong expression of our genes in <i>N. benthamiana</i> leaves. Transcription of the ta-siRNA targeting the WUSCHEL gene is under the control of a light-inducible promoter, which consists of the mphR(A) DNA-binding site, (etr)<sub>8</sub>, followed by the 35S CaMV core promoter. Additionally, we inserted a neomycin resistance cassette into the UVR8-COP1 containing plasmid and a hygromycin resistance cassette in the ta-siRNA containing plasmid near the LB-T-DNA repeats, so that we could select for the final transformed plant in the end.
                    </p>
                    <p style="font-size: 18px;">
                      For pCOP1-UVR8, a total of 3 inserts were cloned into a pFGL815 vector: a neomycin resistance cassette, a 35S CaMV promoter-COP1(WD40)-L-VP16-T2A insert, and TetR-L-UVR8-35S CaMV terminator insert. The COP1-VP16 and TetR-UVR8 fusion proteins were interrupted by a self-cleaving T2A peptide, allowing both fusion proteins to be translated from a single transcript. In an attempt to improve expression of these recombinant proteins, the coding sequences were codon optimized for <i>N. benthamiana</i>. The inserts were oriented such that the 35S promoter was away from the LB T-DNA repeat to prevent degradation of our construct inside the plant tissue; the LB T-DNA repeat is prone to deletion. Furthermore, the neomycin resistance cassette was reversed to control against the possibility of transcription errors.
                    </p>
                    <br>
                    <div align="center">
                    <img src="https://2020.igem.org/wiki/images/2/2e/T--Stony_Brook--tasi.png" style="width:100%; padding-right:10px; padding-bottom:10px;">
                    </div>
                    </div>

                    <div class="text-container" style="display: inline-block;">
                        <p style="font-size: 35px;">
                            References
                        </p>
                        <hr class="hr">
                    <div style="padding-left: 4em; text-indent: -4em;">
                        <p> Adibi, M., Yoshida, S., Weijers, D., & Fleck, C. (2016). Centering the Organizing Center in the Arabidopsis thaliana Shoot Apical Meristem by a Combination of Cytokinin Signaling and Self-Organization. Plos One, 11(2). doi:10.1371/journal.pone.0147830 </p>
                        <p> Allen, E., & Howell, M. D. (2010). MiRNAs in the biogenesis of trans-acting siRNAs in higher plants. Seminars in Cell & Developmental Biology, 21(8), 798-804. doi:10.1016/j.semcdb.2010.03.008 </p>
                        <p> Carpenter, J. E. (2011). Impact of GM crops on biodiversity. GM Crops, 2(1), 7–23. doi: 10.4161/gmcr.2.1.15086 </p>
                        <p> Carthew, R. W., & Sontheimer, E. J. (2009). Origins and Mechanisms of miRNAs and siRNAs. Cell, 136(4), 642-655. doi:10.1016/j.cell.2009.01.035 </p>
                        <p> Fletcher J. C. (2018). The CLV-WUS Stem Cell Signaling Pathway: A Roadmap to Crop Yield Optimization. Plants (Basel, Switzerland), 7(4), 87. https://doi.org/10.3390/plants7040087 </p>
                        <p> Ochoa-Fernandez, R., Abel, N. B., Wieland, F.-G., Schlegel, J., Koch, L.-A., Miller, J. B., Engesser, R., Giuriani, G., Brandl, S. M., Timmer, J., Weber, W., Ott, T., Simon, R., & Zurbriggen, M. D. (2020). Optogenetic control of gene expression in plants in the presence of ambient white light. Nature Methods, 17(7), 717–725. https://doi.org/10.1038/s41592-020-0868-y </p>
                        <p> Somssich, M., Je, B. I., Simon, R., &; Jackson, D. (2016). CLAVATA-WUSCHEL signaling in the shoot meristem. Development, 143(18), 3238-3248. doi:10.1242/dev.133645</p>
                        <p> Yang, F., & Naqvi, N. I. (2014). Sulfonylurea resistance reconstitution as a novel strategy for ILV2-specific integration in Magnaporthe oryzae. Fungal Genetics and Biology, 68, 71-76. doi:10.1016/j.fgb.2014.04.00 </p>
                        <p> Yang, X., Montano, S., & Ren, Z. (2015). How Does Photoreceptor UVR8 Perceive a UV-B Signal?. Photochemistry and photobiology, 91(5), 993–1003. https://doi.org/10.1111/php.12470 </p>
                    </div>
                    </div>
                    <br>
                    <br>

        <div class = "Social_Media">
        </br>
        </br>
        </br>
        <h1>
            <u>
                Stony Brook iGEM 2020
            </u>
        </h1>
        </br>

        <a href = "https://www.instagram.com/igem.stonybrook/?hl=en" target = "_blank">
            <img src = "https://2020.igem.org/wiki/images/a/a3/T--Stony_Brook--instagram.png" class = "social-padding"/>
            </a>

            <a href = "https://www.facebook.com/iGEMatstonybrook/" target = "_blank">
            <img src = "https://2020.igem.org/wiki/images/9/96/T--Stony_Brook--face.png" class = "social-padding" />
            </a>
            <a href = "mailto:igem.sbu@gmail.com" target = "_blank" >
            <img src = "https://2020.igem.org/wiki/images/2/26/T--Stony_Brook--mail.png" class = "social-padding" />
            </a>
            <a href = "https://twitter.com/igemsbu?lang=en" target = "_blank">
            <img src = "https://2020.igem.org/wiki/images/c/cf/T--Stony_Brook--twitter.png" class = "social-padding" />
            </a>



        </br>
        </br>
        </br>
        </br>
        </br>
        </div>
        <script src = "assets/js/hover.js"></script>
    </body>
</html>
